packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -208,6 +209,15 @@ class FlutterDevice {
           getSkSLMethod: getSkSLMethod,
           device: device,
         );
+        await service.streamListen(EventStreams.kExtension);
```

Comment:
This should probably be in a try/catch, in case something else was already listening.

Reply:
It was in the try/catch with `connectToVmService` but probably deserves its own try/catch. So it sounds like if something else is already listening for extension events, then it won't be possible to also attach this listener? If that's the case then maybe the next line (the event processing) should also be in this try/catch too.